charts/audittail/templates/_values.tpl

```diff
@@ -9,7 +9,7 @@ audittail log path
 
 
 {{- define "audittail.image" -}}
-{{- printf "ghcr.io/metal-toolbox/audittail:v0.5.1" -}}
+{{- printf "ghcr.io/metal-toolbox/audittail:v0.1.7" -}}
```

Comment:
Why is this back to using an older release?

Reply:
sorry 👎 updating now :-(

Reply:
It's my bad..updated it new version and ran the pipeline.